.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
[required] kindly remove this directory `.vscode` is not needed by your project, so letting it in your project, makes it to look unprofessional.